db/models/review.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Review = db.model({
+	body: {
+		type: Sequelize.TEXT,
+		allowNull: false,
+	}, 
+	rating: {
+		type: Sequelize.INTEGER.UNSIGNED,
+		allowNull: false,
+		validate: {
+			min: 1,
+			max: 5,
+		},
+	},
+	date: {
```

Comment:
Do we need to have a date column or could we use the one created by Sequelize (or Postgres?) by default? (createdAt/updatedAt)